adrs/php/0002_compiler_module.md

```diff
@@ -0,0 +1,34 @@
+# Compiler Module
+
+## Motivation
+
+Describe the responsibilities of the compiler module.
+
+## Compiler Submodules
+
+- Lexer
+- Parser
+- Reader 
+- Analyzer
+- Emitter
```

Comment:
We now have a Evaluator.

### Evaluator

The evaluator takes the generated PHP String and evaluates it.

Reply:
Done